src/coreclr/jit/morph.cpp

```diff
@@ -17,6 +17,13 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 
 #include "allocacheck.h" // for alloca
 
+#undef min
+#undef max
+
+#if TARGET_WASM
+#include "llvm.h"
+#endif // TARGET_WASM
```

Comment:
Would redefining `min`/`max` after the `#include` work here? Seems better to avoid these small divergences with upstream.